pages/about.html

```diff
@@ -104,20 +103,7 @@ <h4>2022</h4>
           </div>
     </section>
 
-    <section class="partner-section">
-      <div class="text-container">
-        <h3>Partner</h3>
-        <hr />
-      </div>
-      <ul class="list-partner">
-        <li id="moz">Mozilla</li>
-        <li>Google</li>
-        <li id="moz">NAVER</li>
-        <li>daumkakao</li>
-        <li>airbnb</li>
-      </ul>
-    </section>
-    <footer>
```

Comment:
- [ ] While making the changes from the previous reviewer, you deleted the _**Partners/Sponsors**_ Section on the mobile version of the _**About**_ Page. The _**Partners/Sponsors**_ section is present in the mobile version but not on the Desktop version. Kindly include the _**Partners/Sponsor**_ section on the mobile version only. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/217817074-5b22f052-6727-46b1-afa1-cb2ae2370eae.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/217817309-c368dc39-2ad1-423d-9a0c-3700def74173.png)